src/Commands/Issue.php

```diff
@@ -109,11 +109,17 @@ private function doExecute(Manager $args) {
         $path = "certs/" . $keyFile . "/" . reset($domains) . "/key.pem";
         $bits = $args->get("bits");
 
-        try {
+        if ($args->get("regenerate-keys")) {
+          $keyPair = (new OpenSSLKeyGenerator)->generate($bits);
+          $keyPair = (yield $keyStore->put($path, $keyPair));
+        }
+        else {
```

Comment:
Please move that to the previous line (`} else {`) and indent the code with four spaces.